src/CLU/Microsoft.CLU/CommandLineParser/UnixCommandLineParser.cs

```diff
@@ -183,17 +183,42 @@ public string FormatParameterName(string name, string type, bool isMandatory, bo
             return builder.ToString();
         }
 
-        private void PresentCommandHelp(ICommandBinder commandBinder, string[] arguments, bool prefix)
+        private void PresentCommandCompletion(string[] arguments)
+        {
+            foreach (var cmd in Help.CommandDispatchHelper.CompleteCommands(CLUEnvironment.GetPackagesRootPath(), arguments).OrderBy((cmd) => cmd.Commandline))
+            {
+                Console.WriteLine(cmd.Commandline);
+            }
+        }
+
+        private void PresentCommandHelp(string[] arguments)
         {
             // BUGBUG - NYI!
             if (arguments.Length == 0)
             {
-                var concatenatedArgs = string.Join(" ", arguments);
-                Console.WriteLine($"Look up help corresponding to the arguments '{concatenatedArgs}'");
+                Console.WriteLine("TODO: Present help for Azure itself...");
```

Comment:
I don't thinkw e should write this to the console: let's put a TODO comment in the code instead